app/controllers/course/experience_points_records_controller.rb

```diff
@@ -9,7 +9,7 @@ class Course::ExperiencePointsRecordsController < Course::ComponentController
 
   def index # :nodoc:
     @experience_points_records =
-      @experience_points_records.active.includes(:updater).order(updated_at: :desc)
+      @experience_points_records.active.includes(:updater, :actable).order(updated_at: :desc)
```

Comment:
Right now, there are N+1 queries. Since only Submissions are awarding points now, it is possible to try to preload all the required items (e.g. assessments so that we can retrieve its name) to avoid the N+1, but it would increase coupling and we might have to do the same for each model that acts_as_experience_points_record. I don't think this is the right way to go.

I'll add pagination in my next PR to mitigate this. Okay to leave the N+1?